net-misc/anydesk/anydesk-5.1.2.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop systemd xdg-utils
+
+DESCRIPTION="Feature rich multi-platform remote desktop application"
+HOMEPAGE="https://anydesk.com"
+SRC_URI="amd64? ( https://download.anydesk.com/linux/${P}-amd64.tar.gz )
+	x86? ( https://download.anydesk.com/linux/${P}-i386.tar.gz )"
+
+# OpeSSL/SSLeay, libvpx, zlib, Xiph, xxHash
+LICENSE="AnyDesk-TOS BSD BSD-2 openssl ZLIB"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/atk
+	dev-libs/glib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/glu
+	media-libs/mesa[X(+)]
+	sys-auth/polkit
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+
```

Comment:
Many of these dependencies are slotted, but specifying slot here is the most important thing. But you should update and check slots for each deps.
Rest looks good to me.

Reply:
Thanks for reviewing. I added slots where it made sense to me.